### ISSUE TITLE
Integrate Web Map that visualizes all Nodes on the network 

### ISSUE THREAD

**Issue by franchyze923** [thenewboston-developers/Website#1074]
Would like to integrate a web map, similar to [this](http://165.22.184.147/) on TNB website. 

Note, that version is incomplete and needs to be reworked. I used something called [Folium](https://python-visualization.github.io/folium/) to create that version and it didn't retain the dynamic aspect of it. I will be going back to just JavaScript. 

At this point, the webmap is a simple static website composed of 3 files, a .html, .js and .css. Everything is done client side. Will need to figure out how to integrate into the existing TNB website. 

Basically hits the TNB Validator endpoint, then loops through the response and feeds the IP address to this service. That service returns the Lat/Long. This service is limited to 45 HTTP requests per minute from an IP address. I'll need to research and see if this is OK or if there are better options for us.

**Comment by fpolignano** [comment]
I can work on this

**Comment by Hristijan95** [comment]
This is a nice idea, but first it has to go trough the Design Team, they'll help make this idea prettier and more useful if accepted.

**Comment by fpolignano** [comment]
sounds good

**Comment by Hristijan95** [comment]
@fpolignano, you can do it. Just make sure it is close to the styling and the design of the website.

**Comment by franchyze923** [comment]
Just made a pull request for this.
